auth0_flutter/README.md

```diff
@@ -1,131 +1,82 @@
+[![License](https://img.shields.io/github/license/auth0/auth0-flutter)](https://www.apache.org/licenses/LICENSE-2.0)
+[![CircleCI](https://img.shields.io/circleci/project/github/auth0/auth0-flutter.svg)](https://circleci.com/gh/auth0/auth0-flutter/tree/main)
 
-Auth0 SDK for Android / iOS Flutter apps.
-
----
-
-## Table of Contents
-
-- [**Features**](#features)
-- [**Documentation**](#documentation)
-- [**Requirements**](#requirements)
-- [**Installation**](#installation)
-- [**Getting Started**](#getting-started)
-  - [Configuration](#configuration)
-  - [Web Auth Configuration](#web-auth-configuration)
-  - [Web Auth Login](#web-auth-login)
-  - [Web Auth Logout](#web-auth-logout)
-  - [Custom Scheme (Android)](#custom-scheme-android)
-  - [SSO Alert Box (iOS)](#sso-alert-box-ios)
-- [**Next Steps**](#next-steps)
-  + [Common Tasks](#common-tasks)
-  + [Web Auth](#web-auth)
-  + [Credentials Manager](#credentials-manager)
-  + [API](#api)
-- [**Advanced Features**](#advanced-features)
-  - [Organizations](#organizations)
-  - [Bot Detection](#bot-detection)
-- [**Issue Reporting**](#issue-reporting)
-- [**What is Auth0?**](#what-is-auth0)
-- [**License**](#license)
-
-## Features
-
-- **Web Auth**
-  + Login
-  + Logout
-  + Automatic storage and renewal of user's credentials
-- **Authentication API**
-  + Login with username or email and password
-  + Signup
-  + Get user information from `/userinfo`
-  + Renew user's credentials
-  + Reset password
-- **Credentials manager**
-- **Support for custom credentials manager implementations**
+<div>
+📚 <a href="#documentation">Documentation</a> - 🚀 <a href="#getting-started">Getting started</a> - 💻 <a href="#api-reference">API reference</a> - 💬 <a href="#feedback">Feedback</a>
+</div>
 
 ## Documentation
 
-- [**Quickstart**](https://auth0.com/docs/quickstart/native/flutter/interactive)
-  <br>Shows how to integrate auth0_flutter into an Android / iOS Flutter app from scratch.
-- [**Sample app**](https://github.com/auth0-samples/auth0-flutter-samples/tree/main/sample)
-  <br>A complete, running Android / iOS Flutter app you can try.
-- [**API documentation**](https://pub.dev/documentation/auth0_flutter/latest/)
-  <br>Documentation auto-generated from the code comments that explains all the available features.
-- [**FAQ**](FAQ.md)
-  <br> Answers some common questions about flutter_auth0.
+- [Quickstart](https://auth0.com/docs/quickstart/native/flutter/interactive) - our interactive guide for quickly adding login, logout and user information to your app using Auth0
+- [Sample app](https://github.com/auth0-samples/auth0-flutter-samples/tree/main/sample) - a full-fledged sample app integrated with Auth0
+- [API documentation](https://pub.dev/documentation/auth0_flutter/latest/) - documentation auto-generated from the code comments that explains all the available features
+- [Examples](EXAMPLES.md) - examples that demonstrate the different ways in which this SDK can be used
```

Comment:
Should we use absolute URLs here to ensure these URLs work on https://pub.dev/packages/auth0_flutter ?

```suggestion
- [Examples](https://github.com/auth0/auth0-flutter/blob/main/EXAMPLES.md) - examples that demonstrate the different ways in which this SDK can be used
```